src/main/java/mate/academy/springboot/datajpa/dto/mapper/CategoryRequestDtoMapper.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+import mate.academy.springboot.datajpa.dto.CategoryRequestDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CategoryRequestDtoMapper implements RequestDtoMapper<CategoryRequestDto, Category> {
```

Comment:
you can have only one mapper for category (or product) and implement several interfaces